src/github.com/getlantern/flashlight/analytics/analytics.go

```diff
@@ -0,0 +1,68 @@
+package analytics
+
+import (
+	"net/http"
+
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/getlantern/analytics"
+	"github.com/getlantern/flashlight/ui"
+	"github.com/getlantern/golog"
+)
+
+const (
+	messageType = `Analytics`
+)
+
+var (
+	log        = golog.LoggerFor("flashlight.analytics")
+	service    *ui.Service
+	httpClient *http.Client
+)
+
+func Configure(autoReport *bool, newClient *http.Client) {
+	httpClient = newClient
+
+	if autoReport != nil && *autoReport {
+		err := Start()
+		if err != nil {
+			log.Errorf("Error starting analytics service: %q", err)
+		}
+	}
```

Comment:
This implies the analytics service should also be shut down (if it's running) and removed from our UI services map. It doesn't look like there's any unregister functionality right now. I'll go ahead and add that?